src/util/getInMemoryPage.test.js

```diff
@@ -11,4 +11,13 @@ describe('getInMemoryPage', () => {
       data: [{n: 'a'}, {n: 'B'}, {n: 'c'}]
     })
   })
+  it('applies a filterFn', () => {
+    const data = [{n: 'c'}, {n: 'B'}, {n: 'a'}]
+    const filter = {sorts: [{id: 'n', isAsc: true}], search: 'B'}
+    const filterFn = (data, search) => data.filter(datum => datum.n === search)
```

Comment:
add tests to test what happens when the filter returns `null` and `undefined`, we should log an error to speed up debugging filter bugs

Reply:
so picky... Whoops, how'd I get here?

Reply:
Someone call security!